content/en/docs/reference/glossary/istiod.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Istiod
+---
+
+The Istiod component is the consolidated monolithic control plane binary that encapsulates the functions of Pilot, Citadel, Mixer, and Galley.
+
+[Learn more about Istiod](/news/releases/1.5.x/announcing-1.5/#introducing-istiod).
```

Comment:
Maybe this would be a better link? https://istio.io/blog/2020/tradewinds-2020/#fewer-moving-parts

Reply:
the upcoming Istiod blog post is the correct link, but it hasn't been published yet. should be out in a day or two.

Reply:
Agreed

Reply:
In that case, should we wait for the new blog post before merging this PR, or merge as is, and update the link in the Istiod blog post PR?
@craigbox WDYT?